plugins/module_utils/redfish_utils.py

```diff
@@ -1597,6 +1597,8 @@ def get_firmware_update_capabilities(self):
 
         data = response['data']
 
+        result['multipart_supported'] = 'MultipartHttpPushUri' in data
```

Comment:
I think it's kind of unique here. (At least from the modules I'm half-way familiar with, which are definitely not all :) )

About whether to keep it here: that's a good question, and it depends a bit on how similar functions in this module_utils act, and in which direction you'd like to go in the future. I think it would also be OK to use `entries` for the bulk of the data, and use other keys to store other data, like the `multipart_supported` here. It also better separates the keys in `entries`, which are taken from actions, from unrelated data like `multipart_supported`.